back/spec/services/local_project_copy_service_spec.rb

```diff
@@ -312,4 +312,243 @@
       end
     end
   end
+
+  if CitizenLab.ee?
+    describe 'when source project has associated content builder layout' do
+      let!(:layout) { create(:layout, code: 'project_description') }
```

Comment:
Are you sure you need the bang here?

Reply:
Nice catch! I thought I did, but I don't!